### DISCUSSION TITLE
Is using the latest version of manaco-editor supported?

### DISCUSSION THREAD

**Discussion by RickeyWard** [suren-atoyan/monaco-react#205] [Q&A] [answered]
The loader has a hardcoded version of `0.22.1` in it for the cnd, but clearly you can change that at load time by setting the path. Is this discouraged? Is this library heavily tested against a particular version of monaco? I write applications for intranet, so I have to assume no internet connectivity and I bundle monaco editor locally.

is it okay to reach for the starts and grab the latest version? Or should I stick with what the loader wants to pull?

**Comment by suren-atoyan** [comment, marked as answer]
Hello @RickeyWard 

each version is tested against a particular version of monaco, but as we are talking about patches and minor versions technically there shouldn't be problems, so it's totally okay to [configure](https://github.com/suren-atoyan/monaco-react#loader-config) the wrapper with your desired version.

```js
import { loader } from "@monaco-editor/react";

loader.config({ paths: { vs: "https://cdn.jsdelivr.net/npm/monaco-editor@0.23.0/min/vs" } });

/*
or, if you load monaco from your local files
install the latest version and

loader.config({ paths: { vs: "path_to_monaco/min/vs" } });
*/
```

**Reply by suren-atoyan** [reply]
@RickeyWard the new version is here `v4.1.0`, with the latest version (`v0.23.0`) of `monaco-editor`
